run_preprocessing.py

```diff
@@ -173,18 +173,18 @@ def preprocess(config: PreprocessingConfig):
     torch.save(val_graph, "data/derived/val_graph.pt")
     torch.save(test_graph, "data/derived/test_graph.pt")
 
-    edges_test = np_groupby_first_col(
-            transactions_test[["customer_id", "article_id"]]
-            .sort_values("customer_id")
-            .to_numpy()
-        )
-    torch.save(edges_test, "data/derived/edges_test.pt")
+    edges_train = np_groupby_first_col(
```

Comment:
I fixed it on `main` (just FYI, you'll probably need to resolve the conflicts)

Reply:
Ok!